README.md

```diff
@@ -8,37 +8,39 @@
 [github-action-image]: https://github.com/CoherentLabs/GameUIComponents/workflows/Run%20Tests/badge.svg
 
 
-This is a suite of custom elements designed specifically for Gameface. All components can also be used in Google Chrome. You can preview them by starting the demo. You can serve the root directory and open the demo.html file using an http-server of your choice. Or use the default setup in the package.
+This is a suite of custom elements designed specifically for [Gameface](https://coherent-labs.com/products/coherent-gameface/). All components can also be used in Google Chrome. You can preview them by starting the demo. You can serve the root directory and open the demo.html file using an http-server of your choice. Or use the default setup in the package.
+All components are npm packages available in the npm registry. Use `npm i coherent-gameface-<component-name>` to install any of them.
+You can also build them from source.
+
+# Running the demo
+
 
 Navigate to the root directory and run:
 
-    npm install
+    `npm install`
 
-This will install a webpack server. After that run:
+This will install a webpack server as well as all other dependencies. After that run:
 
-    npm run build
+    `npm run build`
 
-This will build all components used in the demo. After that run:
+This will build all components. After that run:
 
-    npm run start:demo
+    `npm run start:demo`
 
 This will serve the files on http://localhost:8080. Load that url in the Gameface player or in Chrome and preview the components.
 You can change the port in the webpack.config.js file.
 
-Custom components examples.
-
-Usage
-===================
 
-Start an http-server in the root of the repository. Set the startup page in Gameface to
-**http://localhost:8080/examples/<example_name>/"** and launch it.
+# Samples
 
-You can use [http-server](https://www.npmjs.com/package/http-server) or any http-server that you like. Make sure you serve all files. The files in **/lib** and **/components** should be accessible.
+The samples are more complex examples of how to create a complete user interface using the
+components. They are located in samples/user_interface. There are three pages - main, settings and shop.
+To run any of them navigate to their folder an run `npm i` to install the dependencies. After that load the
+*.html file in Chrome. Keep in mind that each page has a link to one of the others and if you haven't installed
+the dependencies there, it will not work as expected. To avoid this make sure you execute `npm i` in all folders.

```

Comment:
Why there are new lines here?

Reply:
I don't like reading long lines so I break them manually.
Removed everywhere that I found unnecessary breaks.

Reply:
The benefit of shorter lines (e.g. using 80 column rule) is that you can make independent comments for relatively short text. Now imagine if there are multiple mistakes in this 96 words paragraph. One would need to make a list of the mistakes or, of course, use the better way of using the **suggestions** feature when reviewing through "File changed". Either way, the comment/suggestion would be relatively big.
Reviewing a code/statement which is of 96 "parts" on a single line would feel a little bit overwhelming and commenting and explaining suggestions on it would be even harder.